vividus/src/main/java/org/vividus/bdd/steps/BddVariableSteps.java

```diff
@@ -239,6 +240,26 @@ public void initVariableWithGivenValues(Set<VariableScope> scopes, String variab
         bddVariableContext.putVariable(scopes, variableName, listOfMaps);
     }
 
+    /**
+     * Loads resource located by path as a byte array and puts this variable into context.
+     * Could be useful for the cases when one needs to pass a variable into step as a binary data.
+     * @param scopes The set (comma separated list of scopes e.g.: STORY, NEXT_BATCHES) of variable's scope<br>
+     * <i>Available scopes:</i>
+     * <ul>
+     * <li><b>STEP</b> - the variable will be available only within the step,
+     * <li><b>SCENARIO</b> - the variable will be available only within the scenario,
+     * <li><b>STORY</b> - the variable will be available within the whole story,
+     * <li><b>NEXT_BATCHES</b> - the variable will be available starting from next batch
+     * </ul>
+     * @param variableName A name of variable to assign the values
+     * @param path to a file
+     */
+    @When("I initialize $scopes variable `$variableName` with content of file from path `$path`")
```

Comment:
maybe 
@When("I initialize $scopes variable `$variableName` with binary content of file from path `$path`")